### ISSUE TITLE
fresh install cant login

### ISSUE THREAD

**Issue by jadeye** [web2project/web2project#274]
Hi

I installed v3.3
Cant login after fresh install (twice!)
Even registered a new account - no go...:-(
Any ideas?

Thanx

**Comment by petermoo** [comment]
I have the same error using the latest download from github. I get the following error. To fix the problem, I replaced '0000-00-00 00:00:00' with CURRENT_TIMESTAMP. There was a change to MySQL near version 5.6.

> Error
SQL query:
CREATE TABLE `sessions` (
  `session_id` varchar(40) NOT NULL default '',
  `session_user` int(10) NOT NULL default '0',
  `session_data` longblob,
  `session_updated` timestamp NOT NULL default CURRENT_TIMESTAMP on update CURRENT_TIMESTAMP,
  `session_created` datetime NOT NULL default '0000-00-00 00:00:00',
  PRIMARY KEY  (`session_id`),
  KEY `session_updated` (`session_updated`),
  KEY `session_created` (`session_created`),
  KEY `session_user` (`session_user`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1

MySQL said: Documentation
#1067 - Invalid default value for 'session_created'

**Comment by petermoo** [comment]
w2pversion was missing. I changed '0000-00-00' to '1000-01-01'
CREATE TABLE `w2pversion` (
  `code_revision` int(10) UNSIGNED NOT NULL DEFAULT '0',
  `code_version` varchar(10) NOT NULL default '',
  `db_version` int(10) NOT NULL default '0',
  `last_db_update` date NOT NULL default '0000-00-00',
  `last_code_update` date NOT NULL default '0000-00-00',
  PRIMARY KEY  (`code_revision`)
) ENGINE=MyISAM DEFAULT CHARSET=latin1;

**Comment by petermoo** [comment]
'1000-01-01' also works for sessions.

**Comment by petermoo** [comment]
There are other tables needing the same change. Projects. tasks.

**Comment by caseysoftware** [comment]
Thanks for the heads up. I'll see about making these changes. It's Mysql 5.6+ that introduces this change, right?

**Comment by petermoo** [comment]
Forums, forum_messages, history, projects, sessions, tasks, task_log, user_access_log, w2version.

The MySQL documentation mentions the new format back to 5.0. I am sure there was no error back then. I do not know when they started rejecting the old format.

**Comment by alc6974** [comment]
I had the same problem, I did a clean installation in WAMPP and XAMMP and I could not log in. When I checked the DB I realized that there were missing tables that were not created between them, the "sessions" in addition to forums, forums,  forum_messages, history, projects, projects, projects, sessions,  tasks, tasks, task_log, task_log, user_access_log, user_access_log, user_access_log, w2pversion.  w2pversion, in the installation folder the file 001_base_install.sql contains them but they are with the value DATETIME DEFAULT '0000-00-00 00:00:00'; and DATETIME NOT NULL DEFAULT. '0000-00-00 00:00:00'; 0000-00-00 is not a valid date format for mysql 5.6+ as indicated in file 056_mysql56_updates.sql. I solve it by creating the tables manually modifying the values ​​in the file 001_base_install.sql located in my installation: C: \ wamp64 \ www \ w2p \ install \ sql, in this way I managed to log in W2P.
